clients/client-python/tests/unittests/auth/test_simple_auth_provider.py

```diff
@@ -50,3 +50,4 @@ def test_auth_provider(self):
             token[len(AuthConstants.AUTHORIZATION_BASIC_HEADER) :]
         ).decode("utf-8")
         self.assertEqual(f"{user}:dummy", token_string)
+        os.environ["GRAVITINO_USER"] = ""
```

Comment:
Why do we need this?

Reply:
I guess to fix this issue: #4136 .

Reply:
@noidname01 How about set it to the origin value, rather than set it to empty string?

Reply:
> @noidname01 How about set it to the origin value, rather than set it to empty string?

Sure, fixed!